docs/category/category.yaml

```diff
@@ -361,3 +361,39 @@ paths:
                 status: 401
                 code: UNAUTHORIZED
                 message: The requested URL requires user authorization.
+  /categories/subjects/names:
```

Comment:
Move this endpoint under the `/categories/{id}/subjects/names` because they're related to each other.